region/brinstar/red.json

```diff
@@ -128,6 +128,29 @@
           "note": "This is meant to contain all Rippers in the top climb except the bottom one"
         }
       ],
+      "enemies": [        
+        {
+          "name": "Geega",
+          "quantity": 2,
+          "homeNodes": [ 1 ]
+        },
+        {
+          "name": "Beetom",
+          "quantity": 1,
+          "homeNodes": null,
+          "betweenNodes": [1,2]
+        },
+        {
+          "name": "Ripper",
+          "quantity": 4,
+          "homeNodes": [ 7 ]
```

Comment:
I think I made the junctions how I did so one Ripper could be at 8 with the other 3 at 7.

It's yet another thing that becomes complicated with the prospect of having a walljump tech.